nifi-nar-bundles/nifi-standard-services/nifi-lookup-services-bundle/nifi-lookup-services/src/main/java/org/apache/nifi/lookup/RestLookupService.java

```diff
@@ -172,6 +173,20 @@ public class RestLookupService extends AbstractControllerService implements Reco
         .addValidator(StandardValidators.TIME_PERIOD_VALIDATOR)
         .build();
 
+    static final AllowableValue PASS_THROUGH = new AllowableValue("Pass-through",
+            "Pass-through","sends successful and unsuccessful HTTP responses to the caller");
+    static final AllowableValue HANDLE = new AllowableValue("Handle Service Errors",
+            "Handle Service Errors", "generates an exception when an unsuccessful HTTP response code is received");
+    public static final PropertyDescriptor PROP_RESPONSE_CODE_HANDLING = new PropertyDescriptor.Builder()
+        .name("rest-lookup-response-code-handling")
+        .displayName("Response Code Handling Strategy")
```

Comment:
On further consideration, what do you think of naming this `Response Handling Strategy` with values of `Returned` and `Evaluated`? Open to additional suggestions, but having a single word could be useful, with the description indicating all responses and response codes are returned, or the response code is evaluated for success or failure based on standard HTTP conventions?

Reply:
The values are fine with me.  I updated the text descriptions...let me know what you think in the latest commit.